service/storage_service.hh

```diff
@@ -843,6 +843,8 @@ public:
     future<> topology_transition();
 
     future<> do_cluster_cleanup();
+    future<> alter_tablets_keyspace(sstring ks_name, std::map<sstring, sstring> replication_options,
+                                    std::optional<service::group0_guard>& guard);
```

Comment:
unused in this patch.
also, is it a free function or a method of `alter_keyspace_statement`?

Reply:
Removd unused declarations.